app/src/NativeBase/Components/List.tsx

```diff
@@ -116,18 +130,65 @@ const List: FC<ListProps> = ({ data, mode, options, searchScreen, type }) => {
       return (
         <FlatList
           data={data as Projects}
-          keyExtractor={project => project.res_id}
           renderItem={({ item }) => {
             return (
-              <ProjectListItem
-                onPress={() => {
-                  navigate('ProjectDetail', { item, key: item.res_id })
-                }}
-              >
-                <ProjectSummary project={item} />
-              </ProjectListItem>
+              <Card margin="2">
```

Comment:
From looking at Figma, the edge of the card should line up horizontally with the edge of the segmented picker above, whereas the card has extra spacing around it at the moment here.  So I think the margin setting here should be changed to `marginY="2"` (i.e. vertical margin only, not horizontal) and then I think it'll line up (though do check).

Reply:
Did not know about X & Y margins! The spacing was driving me mad 😄